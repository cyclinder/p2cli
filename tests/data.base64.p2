toBase64
{{decoded|toBase64}}

fromBase64
{{encoded|fromBase64}}
