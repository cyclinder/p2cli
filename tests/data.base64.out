toBase64
c29tZSB2YWx1ZQ==

fromBase64
some value
